search-commons/src/main/java/no/unit/nva/search2/common/OpenSearchSwsClient.java

```diff
@@ -65,11 +64,7 @@ public OpenSearchSwsResponse doSearch(URI requestUri, String mediaType)  {
     private OpenSearchSwsResponse handleResponse(HttpResponse<String> response) throws BadGatewayException {
         if (response.statusCode() != HttpStatus.SC_OK) {
             logger.error(response.body());
-            var problem =
-                attempt(() -> getResponseBodyAs(response, ProblemResponse.class))
-                    .or(() -> new ProblemResponse(response.body(),null,null,null,null, null))
-                    .get();
-            throw new BadGatewayException(problem.detail());
+            throw new BadGatewayException(response.body());
```

Comment:
we return the internal response to client?

Reply:
I will fix it on monday, I promise